src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Primitives/ComposablePartException.cs

```diff
@@ -136,11 +135,8 @@ public ComposablePartException(string message, ICompositionElement element, Exce
         protected ComposablePartException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            _element = info.GetValue<ICompositionElement>("Element");
```

Comment:
Isn't this and the GetObjectData required to make serialization work? We should not delete those on any exceptions.

Basically we want to retain serialization support for exceptions, so keep anything serialization related in those.

Reply:
Same for other exceptiosn eg CompositionException lost some serialization bits.

Reply:
Note this means you will need some FEATURE_SERIALIZATION blocks in types that are fields of exceptions. for example the implementation for ICompositionElement.ToSerializableElement and other random blocks eg the one in SerializableCompositionElement

Reply:
Before we call this port done, please check with Viktor about how to update src\System.Runtime.Serialization.Formatters\tests\BinaryFormatterTestData.cs to make sure these  exceptions are serializing okay.